charts/keycloak/values.yaml

```diff
@@ -74,6 +74,9 @@ keycloak:
     #   exec:
     #     command: ["/bin/sh", "-c", "ls"]
 
+  ## Override the default for the Keycloak container, e.g. for clusters with large cache that requires rebalancing.
+  terminationGracePeriodSeconds: 60
```

Comment:
We should stick with the default of 30.

Reply:
I believe changing the default value would be a breaking change that warrants a major version bump and a note in the upgrade guide to warn users of the change. Are you happy for me to do this as well?

If you don't mind my asking, are you running a cluster with the defaults in this chart and don't have any issues rolling the cluster? e.g. infinispan replication errors, loss of sessions and/or users getting logged out?

Reply:
But it is not changing the default value, only add the possibility to use another value.
IMHO it is not a breaking change, because `terminationGracePeriodSeconds` will keep as `60` for users that are not changing it.

Reply:
I figured we already had 60 before. All good. 😉